projects/1.20/assets/rats/rats/lang/zh_cn.json

```
@@ -0,0 +1,1028 @@
+  "advancement.ratlantis.marbled_cheese_golem_core.title": "我们可以重构……",
+  "advancement.ratlantis.neoratlantean.desc": "击败新·鼠西族",
+  "advancement.ratlantis.neoratlantean.title": "但伙计，我数学不好吗",
+  "advancement.ratlantis.oratchalcum_ingot.desc": "获得鼠铜锭",
+  "advancement.ratlantis.oratchalcum_ingot.title": "感受力量！",
+  "advancement.ratlantis.pirat.desc": "遇到鼠盗，它是一只放荡不羁、放浪形骸的老鼠",
+  "advancement.ratlantis.pirat.title": "准备好了吗？孩子们",
+  "advancement.ratlantis.rat_upgrade_archeologist.desc": "合成老鼠升级：考古学家，鼠西洲的备选方案",
+  "advancement.ratlantis.rat_upgrade_archeologist.title": "温新知故",
+  "advancement.ratlantis.rat_upgrade_buccaneer.desc": "合成老鼠升级：海盗，为老鼠提供了用不着的16世纪武装",
+  "advancement.ratlantis.rat_upgrade_buccaneer.title": "是的船长！",
+  "advancement.ratlantis.rat_upgrade_mount_biplane.desc": "合成老鼠升级：双翼机坐骑，然后获得你自己的王牌飞行员老鼠",
+  "advancement.ratlantis.rat_upgrade_mount_biplane.title": "霍鼠德·休斯",
+  "advancement.ratlantis.rat_upgrade_nonbeliever.desc": "合成老鼠升级：非信者，可以完爆一切的老鼠升级",
+  "advancement.ratlantis.rat_upgrade_nonbeliever.title": "对于非信者来说战神算什么？",
+  "advancement.ratlantis.rat_upgrade_psychic.desc": "合成老鼠升级：灵媒，并且拥有自己的通灵鼠",
+  "advancement.ratlantis.rat_upgrade_psychic.title": "才不是U盘",
+  "advancement.ratlantis.rat_upgrade_ratinator.desc": "合成老鼠升级：终结鼠，将致命的赛博老鼠变为现实",
+  "advancement.ratlantis.rat_upgrade_ratinator.title": "如同泪水消失在雨中",
+  "advancement.ratlantis.ratglove_petals.desc": "利用九个鼠套花合成鼠套花瓣",
+  "advancement.ratlantis.ratglove_petals.title": "花的力量",
+  "advancement.ratlantis.ratlantean_ratbot.desc": "遭遇鼠西族机器鼠，旧时代的遗物。",
+  "advancement.ratlantis.ratlantean_ratbot.title": "不锈钢老鼠",
+  "advancement.ratlantis.ratlantean_spirit.desc": "遇到湮没已久的鼠西族鬼魂",
+  "advancement.ratlantis.ratlantean_spirit.title": "一兆死灵遗灰",
+  "advancement.ratlantis.ratlantis.desc": "使用肥厚奶酪令牌在地上随处创建通往鼠西州的入口。并通过入口进入！",
+  "advancement.ratlantis.ratlantis.title": "鼠西洲！",
+  "advancement.ratlantis.ratlantis_armor.desc": "获得一整套的鼠西族战甲",
+  "advancement.ratlantis.ratlantis_armor.title": "贴金大佛！",
+  "advancement.ratlantis.root.desc": "一个失落的帝国",
+  "advancement.ratlantis.root.title": "鼠西洲",
+  "advancement.ratlantis.token.desc": "从老鼠身上获得大块奶酪令牌！做得好！",
+  "advancement.ratlantis.token.title": "抽卡！十连！全是SSR！！",
+  "advancement.ratlantis.upgrade_combiner.desc": "合成老鼠升级组合器，并开始将这些升级混合在一起",
+  "advancement.ratlantis.upgrade_combiner.title": "最  大  出  力！！！",
+  "advancement.ratlantis.vial_of_sentience.desc": "使用鼠西族魂火和野鼠爪制造知性小瓶。确保将其扔向野蛮鼠西族！",
+  "advancement.ratlantis.vial_of_sentience.title": "我冒的险是经过计算的……",
+  "advancement.rats.all_hats.desc": "收集所有帽子！",
+  "advancement.rats.all_hats.title": "帽子收藏家",
+  "advancement.rats.ball_of_filth.desc": "用大量的垃圾制作一个污渍球",
+  "advancement.rats.ball_of_filth.title": "洁净近乎圣洁！",
+  "advancement.rats.black_death.desc": "遭遇黑死神，是一名被雷击召唤的着魔瘟疫医生",
+  "advancement.rats.black_death.title": "群魔乱舞！",
+  "advancement.rats.cheese.desc": "将一块奶酪变成四个奶酪",
+  "advancement.rats.cheese.title": "我们忘拿饼干了！",
+  "advancement.rats.cheese_stick.desc": "合成奶酪魔杖，用于设置老鼠的存放标记",
```

```suggestion
  "advancement.rats.cheese_stick.desc": "合成奶酪魔杖，用于设置老鼠的存取标记",
```
drop-off and pickup 下同